src/module_wrap.cc

```diff
@@ -622,6 +628,41 @@ void ModuleWrap::SetImportModuleDynamicallyCallback(
   iso->SetHostImportModuleDynamicallyCallback(ImportModuleDynamically);
 }
 
+void ModuleWrap::HostInitializeImportMetaObjectCallback(
+    Local<Context> context, Local<Module> module, Local<Object> meta) {
+  Isolate* isolate = context->GetIsolate();
+  Environment* env = Environment::GetCurrent(context);
+  ModuleWrap* module_wrap = ModuleWrap::GetFromModule(env, module);
+
+  if (module_wrap == nullptr) {
+    env->ThrowError("ModuleWrap object not found in map");
```

Comment:
How about we just return and leave the object empty? The upcoming `vm.Module` PR will make it possible to have Modules that are not in the core module map.

Reply:
Can't the `vm.Module` code just delete the property if it is configurable?

Reply:
Yes, but this line throws an error. We don’t want that.

Reply:
I will remove the throw, but we will probably have to come up with a solution to configure `import.meta` in `vm.Module`